### ISSUE TITLE
hazelcast shell or minitorWar should be enhanced.

### ISSUE THREAD

**Issue by bwzhang2011** [hazelcast/hazelcast#5541]
hi, hazelcast team. when test with hazelcast with its deployment, we switch to bin directory and execute server script to start hazelcast instance and we switch to mancenter directory and execute starManCenter script to start mancenter war. but It lack the support to stop hazelcast instance in server script (by the way, it's very strange that I could not find the procId with hazelcast but I can use netstat to find the port occupied by hazelcast instance) and for mancenter war, it default try to start in 8080 port but such port is often occupied by other web container and also it does not provide the way to change the default port.  hope someone encountered such problem could leave me some message and hz team could add some enhancement for next version release.

**Comment by pveentjer** [comment]
Why don't you write your own scripts? These scripts are just toy scripts; they are not meant to be used in production.

**Comment by emrahkocaman** [comment]
You can specify port number and context root when using `startManCenter` script.
i.e: `./startManCenter.sh 8080 mancenter`. 
Please see [manual](http://docs.hazelcast.org/docs/3.5/manual/html-single/hazelcast-documentation.html#management-center) for other installation options.

Too see processId of hazelcast instance, you may use below script
`ps -a |grep StartServer`

Beyond that, I agree with @pveentjer , if you want more control about your instances you should use your own scripts.

**Comment by bwzhang2011** [comment]
@emrahkocaman, thanks for view and much appreciate of your help and thanks all the same with @pveentjer. as you suggestion, I can start mancenter with different port and sorry I don't carefully read the manual. for the second one. I could not find the the hazelcast instance but I found some info that's very strange. 
first, I use ps -ef|grep  StartServer —— I could not find any procId with hazelcast instance
second I use netstat -anp|grep 5701 I found connection from hazelcast like follows:
![Uploading 1111.png…]()
